examples/hard-contradiction-type-validation-1/profile.json

```diff
@@ -1,41 +0,0 @@
-{
```

Comment:
This profile (and the other type contradiction one) are referenced in https://github.com/finos/datahelix/blob/b75962f4e9caa97970ece405bdeaa3f38e7e9d51/docs/archive/user/Contradictions.md, so the references there needed to be removed.

Reply:
Do we need to worry about these files as they are archived documentation?

Reply:
Ah, probably not then!